### ISSUE TITLE
Unable to use NOC syntax for /etc/rvd.conf

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/rvc#245]
My install is from homebrew-rvc.

My `/opt/rvc/etc/rvd.conf`

```hcl
global {
  user_id = 501;
  restrict_socket = true
  log_directory = "/var/log/rvd"
  vpn_config_paths = "/opt/rvc/etc/vpn.d";
}

openvpn {
  sbin_path = "/opt/openvpn/sbin/openvpn"
  root_check = true
  enable_updown_scripts = false
}
```

When I start `rvd` with it I get this:
```sh
$ rvd -f /opt/rvc/etc/rvd.conf 
Couldn't parse json configuration 'global {
  user_id = 501;
  restrict_socket = true
  log_directory = "/var/log/rvd"
  vpn_config_paths = "/opt/rvc/etc/vpn.d";
}

openvpn {
  sbin_path = "/opt/openvpn/sbin/openvpn"
  root_check = true
  enable_updown_scripts = false
}
' from config file
```

**Comment by ronaldtse** [comment]
The content of the `/opt/rvc/etc/rvd.conf` file is almost identical as https://github.com/riboseinc/rvc/blob/master/etc/rvd.conf.in .

**Comment by jjr840430** [comment]
@ronaldtse Fixed by #250
